src/styles/blocks/filters.scss

```diff
@@ -0,0 +1,28 @@
+.filters {
+  position: absolute;
+  right: 0;
+  left: 0;
+
+  @media (max-width: 430px) {
```

Comment:
Consider using a variable for the media query breakpoint (e.g., $mobile-breakpoint) to keep media queries consistent across the project.

Reply:
Done.